src/Microsoft.AspNet.Mvc.Core/ModelBinding/CompositeValueProvider.cs

```diff
@@ -30,8 +30,8 @@ public CompositeValueProvider()
         /// </summary>
         /// <param name="valueProviders">The sequence of <see cref="IValueProvider"/> to add to this instance of
         /// <see cref="CompositeValueProvider"/>.</param>
-        public CompositeValueProvider(IEnumerable<IValueProvider> valueProviders)
-            : base(valueProviders.ToList())
+        protected CompositeValueProvider(IList<IValueProvider> valueProviders)
+            : base(valueProviders)
```

Comment:
We have product code that updates the `CompositeValueProvider`. I don't see much reason to change this detail or the usage pattern in that part of the code